equipment-models/src/main/java/com/telecominfraproject/wlan/equipment/models/events/EquipmentChangeType.java

```diff
@@ -10,11 +10,11 @@
 
 public enum EquipmentChangeType {
 
-    All(0), ChannelsOnly(1), CellSizeAttributesOnly(2), UNSUPPORTED(-1);
+    All(0), ChannelsOnly(1), CellSizeAttributesOnly(2), ApImpacting(3), UNSUPPORTED(-1);
 
     private final int id;
     private static final Map<Integer, EquipmentChangeType> ELEMENTS = new HashMap<>();
-    private static final EquipmentChangeType validValues[] = { All, ChannelsOnly };
+    private static final EquipmentChangeType validValues[] = { All, ChannelsOnly, CellSizeAttributesOnly, ApImpacting};
```

Comment:
Not sure how to structure or rename this but ChannelsOnly and CellSizeAttributesOnly are also examples of changes which impact the AP.